packages/analytics-js-integrations/src/integrations/Gainsight_PX/browser.js

```
@@ -0,0 +1,119 @@
+
+      const accountObj = {
+        id: this.analytics.getGroupId(),
+        ...this.analytics.getGroupTraits(),
+      };
+
+      window.aptrinsic('identify', visitorObj, accountObj);
+    }
+  }
+
+  isLoaded() {
+    return !!(window.aptrinsic && window.aptrinsic.init);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)